src/madhavi_Khasbage/Coding_Exam_2/SumOfInteger.java

```diff
@@ -0,0 +1,37 @@
+package madhavi_Khasbage.Coding_Exam_2;
+
+/**
+ * Coding Exam - 2 : 26th April 2021 Program 1 : WAP to do sum of all the
+ * integer from the given string Input: String str = "12h14i4w8sdc15" Output:
+ * 12+14+4+8+15 = 53
+ */
+public class SumOfInteger {
+
+	int getSumOfInteger(String strInput) {
+		int calSum = 0, num = 0;
+		String strData = "";
+		for (int index = 0; index < strInput.length(); index++) {
+			char ch = strInput.charAt(index);
+			if (Character.isDigit(ch)) {
+				strData += ch;
+			} else {
+				if (strData != "") {
+					num = Integer.parseInt(strData);
+				}
+				calSum += num;
+				num = 0;
+				strData = "";
+			}
+			if ((index + 1 == strInput.length()) && strData != "") {
```

Comment:
You can remove this condition and outside the loop you can write,

if (temp != "") { //15
	sum += Integer.parseInt(temp); //38+15 = 53
}